### PR TITLE
Add block book servers

### PR DESCRIPTION
**Pull request by TheCharlatan**
Fixup to the previous pull request

### CODE REVIEW

File: src/common/plugin/pluginState.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class PluginState extends ServerCache {
     // TODO: Change the below line to blockbook once available
```
Comment (samholmes) [inline src/common/plugin/pluginState.ts:99]: Should this line be removed now?

**Review by samholmes — APPROVED** [review]
Small nit if applicable.

File: src/common/plugin/pluginState.ts (hunk 1 above)
Comment (TheCharlatan) [reply src/common/plugin/pluginState.ts]: i removed it.

Comment (samholmes) [reply src/common/plugin/pluginState.ts]: Looks good. Apply the fixup commit, and then merge.

**Pull request merged**